parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoMessageConverter.java

```diff
@@ -22,7 +22,10 @@
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.Message;
 import com.twitter.elephantbird.util.Protobufs;
+import org.apache.commons.lang.StringUtils;
```

Comment:
There is no direct dependency to `commons-lang` in `parquet-protobuf` so I would not start using it for such a simple functionality.